### ISSUE TITLE
Terminate mpv on main process quit

### ISSUE THREAD

**Issue by notpushkin** [Songbee/desktop#7]
The process is stored in `player.mpvPlayer` attribute:
```js
>>> _backend.player.mpvPlayer
ChildProcess {domain: null, _events: EventHandlers, _eventsCount: 2, …}
>>> _backend.player.mpvPlayer.kill("SIGTERM")
true
```

However, sometimes `.kill()` doesn't work for some reason.

There's also the `quit` mpv command.

**Comment by iamale** [comment]
The process is stored in `player.mpvPlayer` attribute:
```js
>>> _backend.player.mpvPlayer
ChildProcess {domain: null, _events: EventHandlers, _eventsCount: 2, …}
>>> _backend.player.mpvPlayer.kill("SIGTERM")
true
```

**Comment by notpushkin** [comment]
Okay, we first need to move mpv to the main process. We also probably don't want to stop playing (and seeding) on the window close, but that's a matter of another issue.

**Comment by notpushkin** [comment]
node-mpv seems to try restarting mpv

```
MPV Player seems to have died. Restarting...
Lost connection to socket. Atemping to reconnect
{ Error: connect ECONNREFUSED /tmp/songbeempv.sock
  … }
Lost connection to socket. Atemping to reconnect
{ Error: connect ECONNREFUSED /tmp/songbeempv.sock
  … }
…
```

@00SteinsGate00, is there any way to cleanly terminate/kill mpv using node-mpv?

**Comment by j-holub** [comment]
Good Point. I think I have also thought about this at some point and also realized, that this is a feature lacking so far. I will implement a way to do so. Don't know how I missed out on that one.

**Comment by j-holub** [comment]
Just wanted to inform you, that I'm still on the matter. I already implemented a method to quit mpv. Now I'm investigating how to properly restart it.
The **spawn** method has no callback when mpv is actually ready. So after restarting it one has to wait for a few miliseconds until it is actually usable.
This might also be solvable with promises.
